rules-java-archives/addon/src/main/java/org/jboss/windup/rules/apps/java/archives/config/ArchiveIdentificationConfigLoadingRuleProvider.java

```diff
@@ -51,19 +60,19 @@ public void visit(File file)
                     {
                         try
                         {
-                            log.info("Loading archive identification metadata from [" + file.getAbsolutePath() + "]");
-                            IdentifiedArchives.addMappingsFrom(file);
+                            log.info("Loading archive identification data from [" + file.getAbsolutePath() + "]");
+                            identifier.addMappingsFrom(file);
                         }
                         catch (Exception e)
                         {
-                            throw new WindupException("Failed to load metadata from file [" + file + "]", e);
+                            throw new WindupException("Failed to load identification data from file [" + file + "]", e);
                         }
                     }
                 };
 
                 FileSuffixPredicate predicate = new FileSuffixPredicate("\\.archive-metadata\\.txt");
-                FileVisit.visit(WindupPathUtil.getUserCacheDir().toFile(), predicate, visitor);
-                FileVisit.visit(WindupPathUtil.getWindupCacheDir().toFile(), predicate, visitor);
+                FileVisit.visit(WindupPathUtil.getUserCacheDir().resolve("nexus-indexer-data").toFile(), predicate, visitor);
```

Comment:
I guess that's OK. No harm and could make it a little clearer (separates it from other potential files in the cache dir..